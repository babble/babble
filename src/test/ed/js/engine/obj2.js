
var o = Object();
o.a = "eliot";
o["b"] = "horowitz";

print( o["a"] + " " + o.b );

o.c = Object();
o.c.d = "food";
print( o["c"]["d"] );
