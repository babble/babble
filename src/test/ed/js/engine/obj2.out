eliot horowitz
food
a
b
c
